superset-frontend/src/assets/stylesheets/superset.less

```diff
@@ -327,6 +313,10 @@ table.table-no-hover tr:hover {
   margin-bottom: 10px;
 }
 
+.m-l-2 {
```

Comment:
This probably got yanked in from another file, and i'm lost in the weeds, but just making sure we're not adding more of these utility classes if we don't need to ;)

Reply:
Yeah, the "global" stylesheets were out of scope of this PR and that `.m-l-2` class looked like it belonged here.